app/models/book_suggestion.rb

```diff
@@ -0,0 +1,6 @@
+class BookSuggestion < ApplicationRecord
+  validates :author, :title, :link, :publisher, presence: true
```

Comment:
Do you like to use url_validator to link attribute? [You can check this](https://github.com/perfectline/validates_url)